lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1717,13 +1717,16 @@ def _generate_normals(self, polygons):
         Generate normals for polygons by using the first three points.
         This normal of course might not make sense for polygons with
         more than three points not lying in a plane.
+
+        Normals point towards the viewer for a face with its vertices in
+        counterclockwise order, following the right hand rule.
         '''
 
         normals = []
         for verts in polygons:
-            v1 = np.array(verts[0]) - np.array(verts[1])
-            v2 = np.array(verts[2]) - np.array(verts[0])
-            normals.append(np.cross(v1, v2))
+            v1 = np.array(verts[1]) - np.array(verts[0])
+            v2 = np.array(verts[0]) - np.array(verts[2])
+            normals.append(np.cross(v2, v1))
```

Comment:
Doesn't really matter, but given that you mention the right-hand rule it seems most natural for me to put verts[0] at my palm, verts[1] at the extremity of my thumb and verts[2] of my index, and then the normal is `np.cross(v1-v0, v2-v0)` (which is equal to what you wrote, but perhaps easier to follow).

Reply:
I was trying to formulate this as a right hand screw rule, but what you have there makes sense too.

Do you want me to change that here, or shall I just do so when I remove this code in #12136

Reply:
Can you change it here, and also the corresponding (vectorized) code in #12136?

Reply:
Done